src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -345,10 +351,8 @@ public void onTraversalStart() {
         logger.info("Callset Map JSON file will be written to " + callsetMapJSONFile);
         logger.info("Importing to array - " + workspace + "/" + GenomicsDBConstants.DEFAULT_ARRAY_NAME);
 
-        // Passing in false here so that sample names will be sorted.
-        // This is needed for consistent ordering across partitions/machines
-        callsetMappingPB = GenomicsDBImporter.generateSortedCallSetMap(new ArrayList<>(sampleNameToVcfPath.keySet()), false);
-
+        //use the given order so this is consistent with the batch importing
```

Comment:
"Pass in true here to use the given ordering, since `sampleNameToVcfPath` is already sorted"